fcrepo-connector-file/pom.xml

```diff
@@ -21,6 +21,12 @@
       <groupId>javax.jcr</groupId>
       <artifactId>jcr</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>org.modeshape</groupId>
+      <artifactId>modeshape-jcr</artifactId>
+      <scope>provided</scope>
```

Comment:
The issue is that a few existing pom.xml files depend on `modeshape-jcr` and only some of them actually define that dependency. Scope `provided` ensures that the given dependency does not propagate transitively. However, thanks to you raising this flag, I see that this PR only works because `fcrepo-http-commons/pom.xml` is still such a perpetrator of transitively leaking the `modeshape-jcr` dependency to the other modules... otherwise, the dependency is never actually `provided` by the container (or fcrepo-webapp), which causes build failure.